opm/input/eclipse/EclipseState/EclipseState.cpp

```diff
@@ -56,6 +57,14 @@
 #include <fmt/format.h>
 
 #include <filesystem>
+#include <map>
+#include <stdexcept>
+#include <string>
+#include <unordered_set>
+#include <utility>
+#include <vector>
+
+#include <stddef.h>
```

Comment:
much work to switch to cstddef ?

Reply:
> much work to switch to cstddef ?

I don't think so, but I'll defer doing so to a separate PR.